pkg/transformers/image_test.go

```diff
@@ -243,7 +243,7 @@ func TestImageTransformer(t *testing.T) {
 		{Name: "gcr.io:8080/my-project/my-cool-app", NewName: "my-cool-app"},
 		{Name: "postgres", NewName: "my-postgres", NewTag: "v3"},
 		{Name: "docker", NewName: "my-docker", Digest: "sha256:25a0d4b4a4c0eb97a1aabb8e29f18e917d05abfe1b7a7c07857230879ce7d3d3"},
-	})
+	}, defaultTransformerConfig.Images)
```

Comment:
By the way, can you add a test with custom image transformer configurations.

Reply:
Sure